{"name": "test_zx4", "status": "broken", "statusDetails": {"message": "AttributeError: 'Login' object has no attribute 'gp_login'", "trace": "driver_fixture = <appium.webdriver.webdriver.WebDriver (session=\"0e7b83d5-90bd-4243-ba0f-6351e0c81c1e\")>\n\n    @pytest.fixture(scope='session',autouse=False)\n    def gp_login_fixture(driver_fixture):\n        csv_file=BASE_DIR + '\\datas\\login.csv'\n        print(\"csv_file:%s\"%csv_file)\n        driver=driver_fixture\n        com=Common(driver)\n        data=com.get_csv_data(csv_file,2)\n        time.sleep(2)\n        trd=First(driver)\n        trd.to_login()\n        logn=Login(driver)\n>       logn.gp_login(data[0],data[1])\nE       AttributeError: 'Login' object has no attribute 'gp_login'\n\nconftest.py:54: AttributeError"}, "start": 1652795334101, "stop": 1652795334101, "uuid": "ebeaba91-46e2-4f5d-96ca-38947e2419cd", "historyId": "3b5318d3ca5e663ff3f9fd73b26e1af7", "testCaseId": "8c723d261790106b637844a675e613a5", "fullName": "testGp.test_new_Gp.TestGp#test_zx4", "labels": [{"name": "tag", "value": "@pytest.mark.usefixtures('back_fixture')"}, {"name": "parentSuite", "value": "testGp"}, {"name": "suite", "value": "test_new_Gp"}, {"name": "subSuite", "value": "TestGp"}, {"name": "host", "value": "Win102021AVUCSJ"}, {"name": "thread", "value": "10404-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "testGp.test_new_Gp"}]}